### ISSUE TITLE
Non-executables files incorrectly marked as executable

### ISSUE THREAD

**Issue by cvoltz** [HewlettPackard/LinuxKI#24]
There are many files in the repository which have their executable bit set even though they are not executable files. This really ugly command will provide a list of the files:
```
fgrep -L '!/bin/bash' $(file $(find . -type f -executable | 
  egrep -v '\.(php|sh)') | egrep -v 'shell|node script' 2>/dev/null | 
  awk '{sub(":", "", $1); print $1}') | sort
```
```
./config/config
./config/config.cmu
./config/config.pdsh
./COPYING.kiinfo
./COPYING.liki
./COPYING.linuxki
./experimental/D3/lib/colorbrewer/colorbrewer.css
./experimental/D3/lib/colorbrewer/colorbrewer.js
./experimental/D3/lib/colorbrewer/LICENSE
./experimental/D3/lib/d3/component.json
./experimental/D3/lib/d3/d3.js
./experimental/D3/lib/d3/d3.min.js
./experimental/D3/lib/d3/.gitignore
./experimental/D3/lib/d3/.gitmodules
./experimental/D3/lib/d3/globals.js
./experimental/D3/lib/d3/index-browserify.js
./experimental/D3/lib/d3/index.js
./experimental/D3/lib/d3/LICENSE
./experimental/D3/lib/d3/Makefile
./experimental/D3/lib/d3/.npmignore
./experimental/D3/lib/d3/package.json
./experimental/D3/lib/d3/README.md
./experimental/D3/lib/d3/.travis.yml
./experimental/D3/lib/divgrid.js
./experimental/D3/lib/firebugx.js
./experimental/D3/lib/jit/LICENSE
./experimental/D3/lib/jquery-1.7.min.js
./experimental/D3/lib/jquery.event.drag-2.2.js
./experimental/D3/lib/jquery.event.drop-2.2.js
./experimental/D3/lib/jquery.jsonp-2.4.min.js
./experimental/D3/lib/jquery.simulate.js
./experimental/D3/lib/jquery.sparkline.min.js
./experimental/D3/lib/jquery-ui-1.8.16.custom.min.js
./experimental/D3/lib/mapper.js
./experimental/D3/lib/parcoord/d3.parcoords.css
./experimental/D3/lib/parcoord/d3.parcoords.js
./experimental/D3/lib/parcoord/.slickgrid.html.swp
./experimental/D3/lib/parcoord/style.css
./experimental/D3/lib/penner/LICENSE
./experimental/D3/lib/polymaps/LICENSE
./experimental/D3/lib/protovis/LICENSE
./experimental/D3/lib/qunit.css
./experimental/D3/lib/qunit.js
./experimental/D3/lib/science/LICENSE
./experimental/D3/lib/science/science.js
./experimental/D3/lib/science/science.lin.js
./experimental/D3/lib/science/science.lin.min.js
./experimental/D3/lib/science/science.min.js
./experimental/D3/lib/science/science.stats.js
./experimental/D3/lib/science/science.stats.min.js
./experimental/D3/lib/slickgrid/controls/slick.columnpicker.css
./experimental/D3/lib/slickgrid/controls/slick.columnpicker.js
./experimental/D3/lib/slickgrid/controls/slick.pager.css
./experimental/D3/lib/slickgrid/controls/slick.pager.js
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_flat_0_aaaaaa_40x100.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_flat_75_ffffff_40x100.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_glass_55_fbf9ee_1x400.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_glass_65_ffffff_1x400.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_glass_75_dadada_1x400.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_glass_75_e6e6e6_1x400.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_glass_95_fef1ec_1x400.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-bg_highlight-soft_75_cccccc_1x100.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-icons_222222_256x240.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-icons_2e83ff_256x240.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-icons_454545_256x240.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-icons_888888_256x240.png
./experimental/D3/lib/slickgrid/css/smoothness/images/ui-icons_cd0a0a_256x240.png
./experimental/D3/lib/slickgrid/css/smoothness/jquery-ui-1.8.16.custom.css
./experimental/D3/lib/slickgrid/examples/example10-async-post-render.html
./experimental/D3/lib/slickgrid/examples/example11-autoheight.html
./experimental/D3/lib/slickgrid/examples/example12-fillbrowser.html
./experimental/D3/lib/slickgrid/examples/example13-getItem-sorting.html
./experimental/D3/lib/slickgrid/examples/example14-highlighting.html
./experimental/D3/lib/slickgrid/examples/example1-simple.html
./experimental/D3/lib/slickgrid/examples/example2-formatters.html
./experimental/D3/lib/slickgrid/examples/example3a-compound-editors.html
./experimental/D3/lib/slickgrid/examples/example3b-editing-with-undo.html
./experimental/D3/lib/slickgrid/examples/example3-editing.html
./experimental/D3/lib/slickgrid/examples/example4-model.html
./experimental/D3/lib/slickgrid/examples/example5-collapsing.html
./experimental/D3/lib/slickgrid/examples/example6-ajax-loading.html
./experimental/D3/lib/slickgrid/examples/example7-events.html
./experimental/D3/lib/slickgrid/examples/example8-alternative-display.html
./experimental/D3/lib/slickgrid/examples/example9-row-reordering.html
./experimental/D3/lib/slickgrid/examples/example-autotooltips.html
./experimental/D3/lib/slickgrid/examples/example-checkbox-row-select.html
./experimental/D3/lib/slickgrid/examples/example-colspan.html
./experimental/D3/lib/slickgrid/examples/example-composite-editor-item-details.html
./experimental/D3/lib/slickgrid/examples/example-custom-column-value-extractor.html
./experimental/D3/lib/slickgrid/examples/example-explicit-initialization.html
./experimental/D3/lib/slickgrid/examples/example-grouping.html
./experimental/D3/lib/slickgrid/examples/example-header-row.html
./experimental/D3/lib/slickgrid/examples/example-multi-column-sort.html
./experimental/D3/lib/slickgrid/examples/example-optimizing-dataview.html
./experimental/D3/lib/slickgrid/examples/example-plugin-headerbuttons.html
./experimental/D3/lib/slickgrid/examples/example-plugin-headermenu.html
./experimental/D3/lib/slickgrid/examples/examples.css
./experimental/D3/lib/slickgrid/examples/example-spreadsheet.html
./experimental/D3/lib/slickgrid/examples/example-totals-via-data-provider.html
./experimental/D3/lib/slickgrid/examples/index.html
./experimental/D3/lib/slickgrid/examples/slick.compositeeditor.js
./experimental/D3/lib/slickgrid/examples/slick-default-theme.css
./experimental/D3/lib/slickgrid/.gitignore
./experimental/D3/lib/slickgrid/images/actions.gif
./experimental/D3/lib/slickgrid/images/ajax-loader-small.gif
./experimental/D3/lib/slickgrid/images/arrow_redo.png
./experimental/D3/lib/slickgrid/images/arrow_right_peppermint.png
./experimental/D3/lib/slickgrid/images/arrow_right_spearmint.png
./experimental/D3/lib/slickgrid/images/arrow_undo.png
./experimental/D3/lib/slickgrid/images/bullet_blue.png
./experimental/D3/lib/slickgrid/images/bullet_star.png
./experimental/D3/lib/slickgrid/images/bullet_toggle_minus.png
./experimental/D3/lib/slickgrid/images/bullet_toggle_plus.png
./experimental/D3/lib/slickgrid/images/calendar.gif
./experimental/D3/lib/slickgrid/images/collapse.gif
./experimental/D3/lib/slickgrid/images/comment_yellow.gif
./experimental/D3/lib/slickgrid/images/down.gif
./experimental/D3/lib/slickgrid/images/drag-handle.png
./experimental/D3/lib/slickgrid/images/editor-helper-bg.gif
./experimental/D3/lib/slickgrid/images/expand.gif
./experimental/D3/lib/slickgrid/images/header-bg.gif
./experimental/D3/lib/slickgrid/images/header-columns-bg.gif
./experimental/D3/lib/slickgrid/images/header-columns-over-bg.gif
./experimental/D3/lib/slickgrid/images/help.png
./experimental/D3/lib/slickgrid/images/info.gif
./experimental/D3/lib/slickgrid/images/listview.gif
./experimental/D3/lib/slickgrid/images/pencil.gif
./experimental/D3/lib/slickgrid/images/row-over-bg.gif
./experimental/D3/lib/slickgrid/images/sort-asc.gif
./experimental/D3/lib/slickgrid/images/sort-asc.png
./experimental/D3/lib/slickgrid/images/sort-desc.gif
./experimental/D3/lib/slickgrid/images/sort-desc.png
./experimental/D3/lib/slickgrid/images/stripes.png
./experimental/D3/lib/slickgrid/images/tag_red.png
./experimental/D3/lib/slickgrid/images/tick.png
./experimental/D3/lib/slickgrid/images/user_identity.gif
./experimental/D3/lib/slickgrid/images/user_identity_plus.gif
./experimental/D3/lib/slickgrid/lib/firebugx.js
./experimental/D3/lib/slickgrid/lib/jquery-1.7.min.js
./experimental/D3/lib/slickgrid/lib/jquery.event.drag-2.2.js
./experimental/D3/lib/slickgrid/lib/jquery.event.drop-2.2.js
./experimental/D3/lib/slickgrid/lib/jquery.jsonp-2.4.min.js
./experimental/D3/lib/slickgrid/lib/jquery.simulate.js
./experimental/D3/lib/slickgrid/lib/jquery.sparkline.min.js
./experimental/D3/lib/slickgrid/lib/jquery-ui-1.8.16.custom.min.js
./experimental/D3/lib/slickgrid/lib/qunit.css
./experimental/D3/lib/slickgrid/lib/qunit.js
./experimental/D3/lib/slickgrid/MIT-LICENSE.txt
./experimental/D3/lib/slickgrid/plugins/slick.autotooltips.js
./experimental/D3/lib/slickgrid/plugins/slick.cellcopymanager.js
./experimental/D3/lib/slickgrid/plugins/slick.cellrangedecorator.js
./experimental/D3/lib/slickgrid/plugins/slick.cellrangeselector.js
./experimental/D3/lib/slickgrid/plugins/slick.cellselectionmodel.js
./experimental/D3/lib/slickgrid/plugins/slick.checkboxselectcolumn.js
./experimental/D3/lib/slickgrid/plugins/slick.headerbuttons.css
./experimental/D3/lib/slickgrid/plugins/slick.headerbuttons.js
./experimental/D3/lib/slickgrid/plugins/slick.headermenu.css
./experimental/D3/lib/slickgrid/plugins/slick.headermenu.js
./experimental/D3/lib/slickgrid/plugins/slick.rowmovemanager.js
./experimental/D3/lib/slickgrid/plugins/slick.rowselectionmodel.js
./experimental/D3/lib/slickgrid/README.md
./experimental/D3/lib/slickgrid/slick.core.js
./experimental/D3/lib/slickgrid/slick.dataview.js
./experimental/D3/lib/slickgrid/slick.editors.js
./experimental/D3/lib/slickgrid/slick.formatters.js
./experimental/D3/lib/slickgrid/slick.grid.css
./experimental/D3/lib/slickgrid/slick.grid.js
./experimental/D3/lib/slickgrid/slick.groupitemmetadataprovider.js
./experimental/D3/lib/slickgrid/slick.remotemodel.js
./experimental/D3/lib/slickgrid/tests/dataview/dataview.js
./experimental/D3/lib/slickgrid/tests/dataview/index.html
./experimental/D3/lib/slickgrid/tests/grid/grid.js
./experimental/D3/lib/slickgrid/tests/grid/index.html
./experimental/D3/lib/slickgrid/tests/index.html
./experimental/D3/lib/slickgrid/tests/plugins/autotooltips.html
./experimental/D3/lib/slickgrid/tests/plugins/autotooltips.js
./experimental/D3/lib/underscore-min.js
./experimental/exp
./experimental/vis/cluster_network.html
./experimental/vis/cluster_timeline.html
./experimental/vis/cluster_timeline.html.orig
./experimental/vis/futex_scatter.html
./experimental/vis/futex_scatter.html.rev2
./experimental/vis/kidsk.html
./experimental/vis/kidsk_scatter.html
./experimental/vis/kidsk_scatter.html.backup
./experimental/vis/kifile.html
./experimental/vis/kipid_io.html
./experimental/vis/kipid_sched.html
./experimental/vis/kirunq.html
./experimental/vis/kiwait.html
./experimental/vis/network.html
./experimental/vis/network.html.save
./experimental/vis/pid_detail.html
./experimental/vis/pid_switch.html
./experimental/vis/pid_switch_msr.awkpgm
./experimental/vis/pid_switch_steal.awkpgm
./experimental/vis/pid_switch_steal_msr.awkpgm
./experimental/vis/pid_wtree.html
./experimental/vis/sample_network.csv
./experimental/vis/server_network.html
./experimental/vis/server_timeline.html
./experimental/vis/socket.html
./experimental/vis/timeline.html
./src/kiinfo/compile
```

**Comment by cvoltz** [comment]
I created pull request #26.

**Comment by MarkCRay** [comment]
Fixed in LinuxKI version 6.0.
